ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ip_access_lists.schema.yml

```
@@ -10,84 +10,96 @@ keys:
     type: list
     primary_key: name
     convert_types:
-    - dict
+      - dict
     display_name: IP Extended Access-Lists (improved model)
     items:
       type: dict
       keys:
         name:
           type: str
-          description: Access-list Name
-          convert_types: [ int ]
+          description: Access-list Name.
+          convert_types:
+            - int
         counters_per_entry:
           type: bool
         entries:
           type: list
-          description: ACL Entries
+          description: ACL Entries.
           items:
             type: dict
             keys:
               sequence:
                 type: int
                 convert_types:
-                - str
-                description: |
-                  ACL entry sequence number.
+                  - str
+                description: ACL entry sequence number.
               remark:
                 type: str
-                description: |
+                description: |-
                   Comment up to 100 characters.
                   If remark is defined, other keys in acl entry will be ignored.
```

While we are fixing the schema..
```suggestion
                  If remark is defined, other keys in ACL entry will be ignored.
```